model/src/channel/message/allowed_mentions/mod.rs

```diff
@@ -0,0 +1,105 @@
+use crate::id::{RoleId, UserId};
+use serde::{Deserialize, Serialize};
+
+mod builder;
+
+pub use self::builder::AllowedMentionsBuilder;
+
+#[allow(clippy::trivially_copy_pass_by_ref)]
+fn is_false(value: &bool) -> bool {
+    !value
+}
+
+/// Parse types.
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "lowercase")]
+pub enum ParseTypes {
+    Users,
+    Roles,
+    Everyone,
+}
+
+/// Allowed mentions structure.
+#[derive(Clone, Debug, Default, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct AllowedMentions {
```

Comment:
This should have an

```rust
impl AllowedMentions {
    pub fn builder() -> AllowedMentionsBuilder { ... }
}
```
to be consistent with https://docs.rs/twilight-model/0.3.5/twilight_model/gateway/payload/request_guild_members/struct.RequestGuildMembersBuilder.html